Tests/Pcap++Test/Tests/LiveDeviceTests.cpp

```diff
@@ -466,6 +466,99 @@ PTF_TEST_CASE(TestPcapLiveDeviceBlockingMode)
 	pcpp::Logger::getInstance().enableLogs();
 } // TestPcapLiveDeviceBlockingMode
 
+PTF_TEST_CASE(TestPcapLiveDeviceBlockingModeWithPoll)
+{
+	// open device
+	pcpp::PcapLiveDevice* liveDev = pcpp::PcapLiveDeviceList::getInstance().getPcapLiveDeviceByIp(PcapTestGlobalArgs.ipToSendReceivePackets.c_str());
+	PTF_ASSERT_NOT_NULL(liveDev);
+
+	pcpp::PcapLiveDevice::DeviceConfiguration defaultConfig;
+	defaultConfig.usePoll = true;
```

Comment:
test with the new flag.

Reply:
I guess we can only test it on supported platforms?

Reply:
I agree. In addition, I would like to incorporate a test to ensure that PCPP can correctly terminate the loop when there are no incoming packets. However, I'm currently unsure of how to implement this within the existing testing framework

Reply:
Yes, I don't have a good idea how to test this scenario, especially in CI when we use docker containers 🤔 

If these were real network interfaces maybe we could use `iptables` to block all network traffic like this: https://serverfault.com/questions/880128/how-to-block-all-traffic-between-two-interfaces

But I don't think this will work in containers...

Reply:
I tried to use a simple way to test it. setting the timeout to only 1ms, and it should get nothing and timeout correctly.

Reply:
Hmm that could work! We just need to make sure it's deterministic. If it's not, maybe we can retry capturing packets a few times until no packets are received

Reply:
@seladb  I don't understand, could you provide an example?

Reply:
the idea about a very short time doesn't work. 
I use iptables to drop all packets for testing this.